pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -428,8 +428,8 @@ let
         mktplcRef = {
           name = "vscode-neovim";
           publisher = "asvetliakov";
-          version = "1.7.1";
-          sha256 = "0ib4sjk7r370ckvaqb4yzgy7csy8pli9z2jhibhhwwcq2748ah4q";
+          version = "1.8.1";
+          sha256 = "0kqzry0cbvvy1pkbx5rhnk5ry7j91cbf11vz1s1jrqc7j0hq0yx8";
```

Comment:
```suggestion
          hash = "0kqzry0cbvvy1pkbx5rhnk5ry7j91cbf11vz1s1jrqc7j0hq0yx8";
```

Reply:
See: https://github.com/NixOS/nixpkgs/pull/258596

Reply:
Also the sha256 hash is as produced by the update script.

Reply:
We are seeking to standardize in `hash`.
Can't delete the hash and it generate a new one? What's the problem?

Reply:
> Also the sha256 hash is as produced by the update script.

Update script needs an update then.